### ISSUE TITLE
Right sidebar toggle button not clickable...

### ISSUE THREAD

**Issue by rubaboo** [pjeby/pane-relief#47]
When enabling sliding panes (for current window), the sidebar toggle button gets relocated to the far right. When in that position, it is not clickable.

Obsidian v1.1.16

**Comment by pjeby** [comment]
Here is a CSS snippet that works around the issue, though it makes it impossible to drag the window when it's in sliding panes mode (unless you grab above the left sidebar):

```css
.workspace.is-right-sidedock-open .mod-root.is-sliding + .mod-right-split .workspace-tabs.mod-top .workspace-tab-header-spacer {
    -webkit-app-region: no-drag;
}
```

I'm trying to figure out a better fix than this, and you should remove this CSS once I do have an official fix.

**Comment by pjeby** [comment]
Ok, an updated fix is in 0.4.2 - if you update you should have a functional button and the drag issue should be resolved as well - you can now get rid of the CSS snippet if you used it.
